x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -235,9 +353,92 @@ Set log level to INFO.
 
 [source,shell]
 ----
-elastic-agent run -v --path.logs path/to/my/logs
+elastic-agent run -c myagentconfig.yml
+----
+
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-uninstall-command]]
+== elastic-agent uninstall
+
+Permanently uninstall {agent} from the system.
+
```

Comment:
A note about being root/Administrator just like `install` command, would be nice.